### PR TITLE
Refactor patient profile display logic to utils

### PR DESCRIPTION
**Pull request by emyl3**
## Related Issue or Background Info
This PR refactors display logic for the patient profile introduced in #622.

## Changes Proposed

- move display logic to `utils`
- write unit tests

## Additional Information

- N/A

## Screenshots / Demos

- N/A

### CODE REVIEW

**Review by nullflux — APPROVED** [review]
Approved in general! ✅ 

Note my one comment on `utils/text.ts`

File: frontend/src/app/utils/text.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const capitalizeText = ({ text = "" }) => {
```
[the remark this answers is not in the document]
Comment (emyl3) [reply frontend/src/app/utils/text.ts]: Thank you for that tip, Adam! Will merge once checks pass. 😸

[the remark this answers is not in the document]
Comment (nullflux) [reply frontend/src/app/utils/text.ts]: Sweet! It should actually be lowercase `string` in there, my bad 🤦‍♂️

[the remark this answers is not in the document]
Comment (emyl3) [reply frontend/src/app/utils/text.ts]: Ah! Gotcha! Thank you, Adam! I looked up the difference between the two in this context. TIL! 👍 Pushed up the change haha :D

**Pull request merged**